dds_cli/__main__.py

```diff
@@ -1286,13 +1289,14 @@ def get_data(
 # Options
 @project_option(required=True)
 @folder_option(help_message="List contents in this project folder.")
+@unit_prefix_option
```

Comment:
I don't think you need this here? 
The unit prefix option is only for the project listing right?